### PR TITLE
Vesion bump in package-lock.json

### PR DESCRIPTION
**Pull request by Boomatang**
# What & Why
Bump the version of the package-lock.json to match package.json.

### CODE REVIEW

**Comment by secondsun** [comment]
@Boomatang You mention updating the package-lock, but you only commited package.json

**Comment by Boomatang** [comment]
@secondsun Thanks for spotting that. My guess guess I somehow dropped when fixing the commit naming. The package-lock should now also be included.

File: package.json [hunk 1]
```diff
@@ -66,4 +66,4 @@
     "react-scripts": "3.4.3",
     "typescript": "3.7.5"
   }
-}
\ No newline at end of file
+}
```
Comment (ziccardi) [inline package.json:69]: Please remove this. The version bump workflow will remove it anyway.

**Review by ziccardi — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Boomatang) [reply package.json]: Newline has been removed

**Review by ziccardi — APPROVED** [review]

**Pull request merged**